doc/model.rst

```diff
@@ -0,0 +1,683 @@
+-  `run_settings` (Type: RunSettings): Describe execution settings for a Model.
+-  `params` (dict, optional): Provides a dictionary of parameters for Models.
+-  `path` (str, optional): Path to where the model should be executed at runtime.
+-  `enable_key_prefixing` (bool, optional): Prefix the model name to data sent to the database to prevent key collisions. Default is True.
+-  `batch_settings` (BatchSettings | None): Describes settings for batch workload treatment.
+
+To initialize a ``Model``, a `name` and :ref:`RunSettings<settings-info>` reference is required.
+To instruct a Model to encapsulate a simulation, users must provide the simulation
+executable to the Models run settings with instructions on how the application should be launched.
+
+Users may also launch Models as a batch job by specifying :ref:`BatchSettings<settings-info>` when initializing.
+When a Model with a ``BatchSettings`` reference is added to an Ensemble with a ``BatchSettings`` reference,
+the Models batch settings are strategically ignored.
+
+The `params` init argument in for Models lets users define simulation parameters and their
+values through a dictionary. Using :ref:`Model API functions<model_api>`, users can write these parameters to
+a file in the Model's working directory. Similarly, the Model API provides functionality to
+instruct SmartSim to read a file and automatically fill in parameter values. Both behaviors are
+provided by the ``Model.attach_generator_files()`` function.
+
+Additionally, it's important to note that Model instances will be executed in the
+current working directory by default if no `path` argument is supplied. When a Model
+instance is passed to ``Experiment.generate()``, a directory within the Experiment directory
+is automatically created to store input and output files from the model.
+
+----------------
+A Standard Model
+----------------
+For standard model deployment, the model runs
+on separate compute nodes from SmartSim orchestrators.
+Standard models are able to communicate with clustered or standalone
+databases. SmartRedis clients connect to a clustered Orchestrator from
+within the Models executable script
+and travel off the simulation compute node to the database compute node to send/retrieve
+data. To use a database within the standard model workload, a
+SmartSim Orchestrator must be launched prior to the start of the Model.
+Standard models are ideal for simulations benefiting from
+distributed computing capabilities, enabling efficient
+parallelized execution across multiple compute nodes.
+
+In the `Instructions` subsection, we provide an example illustrating the deployment of a standard model.
+
+Instructions
+------------
+This example provides a demonstration of how to initialize and launch a standard Model
+within the Experiment workflow.
+
+All workflow entities are initialized through the Experiment API. To create a Model,
+the function ``Experiment.create_model()`` is used. To create Model, we must first
```

Comment:
"To create Model, we must first" is probably better as "To create a|the Model..."

Reply:
aye I reworded it to rid of the ,